src/app/rundown-view/components/rundown-header-context-menu/rundown-header-context-menu.component.ts

```diff
@@ -65,6 +99,14 @@ export class RundownHeaderContextMenuComponent {
     this.rundownService.activate(this.rundown.id).subscribe()
   }
 
+  public openRehearseRundownDialog(): void {
```

Comment:
Here the verb of the method is `open` and what `open` refers to is a noun, so it should be `Rehearsal`.

Reply:
Fixed